src/math/bvh.js

```diff
@@ -0,0 +1,456 @@
+import { Vec3 } from '../math/vec3.js';
+import { BoundingBox } from '../shape/bounding-box.js';
+
+/** @typedef {import('../shape/ray.js').Ray} Ray */
+/** @typedef {import('../math/tri.js').Tri} Tri */
+
+class BVHNode {
```

Comment:
mark whole class with `@ignore`, instead of individual functions.

Reply:
Marked all classes in this file as `@ignore`